extension.json

```
@@ -1,6 +1,6 @@
 {
   "name": "FlexForm",
-  "version": "2.5.0",
+  "version": "2.5.5",
```

```suggestion
  "version": "2.6.0",
```

Semantic versioning.